[DOCS] Adds outlier detection params to the data frame analytics resources (#46323)
elastic/elasticsearch: docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

An {oldetection} configuration object has the following properties:

`compute_feature_influence`::
  (boolean) If `true`, the feature influence calculation is enabled. Defaults to 
  `true`.
  
`feature_influence_threshold`:: 
  (double) The minimum {olscore} that a document needs to have in order to 
  calculate its {fiscore}. Value range: 0-1 (`0.1` by default).

`method`::
  (string) Sets the method that {oldetection} uses. If the method is not set 
  {oldetection} uses an ensemble of different methods and normalises and 
  combines their individual {olscores} to obtain the overall {olscore}. We 
  recommend to use the ensemble method. Available methods are `lof`, `ldof`, 
  `distance_kth_nn`, `distance_knn`.
  
`n_neighbors`::
  (integer) Defines the value for how many nearest neighbors each method of 
  {oldetection} will use to calculate its {olscore}. When the value is 
  not set, the system will dynamically detect an appropriate value.
  
`outlier_fraction`::

>   (double) Sets the proportion of data you assume that is outlier prior to the 

Author: ```suggestion
  (double) Sets the proportion of data that is assumed to be outlier prior to the 
```

>   (double) Sets the proportion of data that is assumed to be outlier prior to the 

Reviewer: ```suggestion
  (double) Sets the proportion of data set that is assumed to be outlying prior to 
```

>   {oldetection}. For example, 0.05 mean you think 5% of values are real outliers 

Author: ```suggestion
  {oldetection}. For example, 0.05 means, it is assumed that 5% of values are real outliers 
```

  and 95% are inliers.
  
`standardize_columns`::

>   (boolean) If `true`, then the following operation is performed on the columns 

Reviewer: One option here would be just to refer to wiki for this, i.e. this link https://en.wikipedia.org/wiki/Feature_scaling#Standardization_(Z-score_Normalization)

An {oldetection} configuration object has the following properties:

`compute_feature_influence`::
  (boolean) If `true`, the feature influence calculation is enabled. Defaults to 
  `true`.
  
`feature_influence_threshold`:: 
  (double) The minimum {olscore} that a document needs to have in order to 
  calculate its {fiscore}. Value range: 0-1 (`0.1` by default).

`method`::
  (string) Sets the method that {oldetection} uses. If the method is not set 
  {oldetection} uses an ensemble of different methods and normalises and 
  combines their individual {olscores} to obtain the overall {olscore}. We 
  recommend to use the ensemble method. Available methods are `lof`, `ldof`, 
  `distance_kth_nn`, `distance_knn`.
  
`n_neighbors`::
  (integer) Defines the value for how many nearest neighbors each method of 
  {oldetection} will use to calculate its {olscore}. When the value is not set, 
  different values will be used for different ensemble members. This helps 
  improve diversity in the ensemble. Therefore, only override this if you are 
  confident that the value you choose is appropriate for the data set.
  
`outlier_fraction`::

>   (double) Sets the proportion of data set that is assumed to be outlying prior to 

Reviewer 2: ```suggestion
  (double) Sets the proportion of the data set that is assumed to be outlying prior to 
```

>   {oldetection}. For example, 0.05 means, it is assumed that 5% of values are real outliers 

Reviewer 2: ```suggestion
  {oldetection}. For example, 0.05 means it is assumed that 5% of values are real outliers 
```
